static/app/utils/theme/theme.chonk.tsx

```diff
@@ -792,6 +798,11 @@ export const DO_NOT_USE_lightChonkTheme: ChonkTheme = {
 
   space,
   radius,
+  focusRing: {
+    outline: 'none',
+    boxShadow: `0 0 0 2px transparent, 0 0 0 4px ${lightAliases.focusBorder}`,
```

Comment:
@JonasBa I don’t think `transparent` works here. It seems to take the color of the focusRing then instead of the background.

Here are before/after images for the input component:

before:

![Screenshot 2025-03-06 at 12 41 03](https://github.com/user-attachments/assets/35d17249-507a-4c3b-a132-2dce761f4e0d)

after:

![Screenshot 2025-03-06 at 12 39 19](https://github.com/user-attachments/assets/2e790b36-766a-4c99-bd7f-296de939daef)